package/yast2-sap-ha.spec

```diff
@@ -12,49 +12,49 @@
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
 
-# Please submit bugfixes or comments via http://bugs.opensuse.org/
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
 Name:           yast2-sap-ha
-Version:        4.5.6
+Version:        4.5.7
 Release:        0
 
 BuildArch:      noarch
 
 Source0:        %{name}-%{version}.tar.bz2
 Source1:        yast2-sap-ha-rpmlintrc
 
+Requires:       conntrack-tools
+Requires:       corosync
+Requires:       corosync-qdevice
+Requires:       crmsh
+Requires:       csync2
+Requires:       hawk2
+Requires:       pacemaker
 Requires:       yast2
+Requires:       yast2-cluster >= 4.4.4
 Requires:       yast2-ruby-bindings
-Requires:       csync2
-Requires:       corosync
+Requires:       yast2-ntp-client
 # for opening URLs
 Requires:       xdg-utils
 # for handling the SSH client
 Requires:       expect
+Requires:       firewalld
 Requires:       openssh
-Requires:       yast2-cluster
-Requires:       yast2-ntp-client
-# for lsblk
+Requires:       HANA-Firewall >= 2.0.3
 Requires:       util-linux
-# lsmod, modprobe
 Requires:       SAPHanaSR
 Requires:       kmod
-# configuration parser
-Requires:       augeas-lenses
-Requires:       rubygem(%{rb_default_ruby_abi}:cfa)
 # for pidof
 Requires:       sysvinit-tools
-# xmlrpc was removed from stdlib
-%if 0%{?suse_version} >= 1540
-Requires:       rubygem(%{rb_default_ruby_abi}:xmlrpc)
-BuildRequires:  rubygem(%{rb_default_ruby_abi}:xmlrpc)
-%endif
 
-BuildRequires:  augeas-lenses
+BuildRequires:  HANA-Firewall >= 2.0.3
```

Comment:
This `HANA-Firewall` and the `firewalld` build dependency below look strange. If you need them because of unit tests then very likely some mocks are is missing...

This also causes failures in the CI: https://github.com/yast/yast-sap-ha/actions/runs/5291105920/jobs/9576174052?pr=33#step:4:45

Running `/usr/bin/firewall-cmd` might also cause troubles in the system which runs the unit tests (i.e. applying some setting) or make the unit tests failing (different than expected firewall configuration or the firewall might not be installed at all).

Reply:
The requirement for HANA-Firewall is necessary because only this version contains the SCR agent for handling the configuration file.
IMHO the unit tests works fine if firewalld is installed.